public/stylesheets/style.css

```diff
@@ -0,0 +1,24 @@
+
+* {
+  border-radius: 0 !important;
```

Comment:
I would like to avoid the use of `!important` - we should instead create more specific styles.